### DISCUSSION TITLE
Pentair Intellichem Dosing pumps--power requirements?

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#714] [REM] [answered]
I think they are 24VDC.

Anyone one know their current draw?  The manual indicates the relay contacts are rated for 5 amps, but the pump current is probably much lower.

Thanks!

**Comment by rstrouse** [comment]
If I were to guess I would say < 1.5A current draw.

**Comment by davthomaspilot** [comment]
I guess I better measure it--I was guessing around .5 amp max.

I already ordered a 2.5A, 24V DIN rail.  It needs to power two pumps simultaneously and provide POE current for a Ubiquiti bullet (high power, outdoor Wifi extender).  

I'm assuming the REM might activate the two pumps simultaneously like Intellichem does?  Though, it would be nice to prevent that in the code.  It's really not necessary to run them at the same time and would cut the power requirements in 1/2 (for REM only case).

**Reply by rstrouse** [reply, marked as answer]
Actually, REM is smarter than that.  Select pH priority and it will drive the pH within range and hold the sanitizer for mixing operations.  This means that both pumps will not engage at the same time.  It also disables chlorine dosing when dosing acid and will hold the acid dose if chlorine is being dosed.

**Reply by davthomaspilot** [reply]
Hey, that's great!

Thanks
